### DISCUSSION TITLE
LinearLayout selector

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#915] [Q&A] [answered]
%m.linearlayout is not working,what's is the placeholder for linear,like %m.textview for textview

**Comment by j-px** [comment, marked as answer]
it doesn't look like a specific selector for that exists because it's never been needed before. can't you just rely on `%m.view`?

**Reply by subhash16** [reply]
Yeah I am using %m.view currently if there is particular for linear then that will be more helpful

**Comment by subhash16** [comment]
.

**Reply by j-px** [reply]
what
